init.lua

```diff
@@ -42,6 +42,7 @@ _M.settings = {
 	client_opacity_value = 0.5,
 
 	cycle_raise_client = true,
+  cycle_all_clients  = true,
```

Comment:
You say in the PR description; "It is disabled by default." but here it is clearly _enabled_.  I think your description is a good default, so please change this and update the README as well.

Also, you seem to use a different tab width setting, so this line does not have the same indentation as surrounding lines.